### DISCUSSION TITLE
Requirement of HuggingFace account and accepting the license

### DISCUSSION THREAD

**Discussion by jordimas** [pyannote/pyannote-audio#1777] [Development]
Hello everybody

First, I want to thanks you for making pyannote-audio open source.

I'm using it in serval open source projects (https://github.com/Softcatala/open-dubbing, https://github.com/Softcatala/whisper-ctranslate2)  and the top complain from end users is the need to create a HuggingFace account, accept the license and get an API key. This make the whole setup for users far more complex and error prone.

Is there is any way to make this easier for users and remove the need to have to create an account in HF, accept the license and get an API key?

There are many open source projects there that combine free and commercial without the need of this complex setup.

Thanks again for your work

Jordi

**Comment by clement-pages** [comment]
Hey @jordimas,

Thank you for using `pyannote`!

`pyannote`'s pipelines and models are gated because it let us who is using `pyannote` and for which purpose. For sure, this implies more steps the first time you are using a pipeline or a model, but this has to be done only the very first time.
